src/const.ts

```diff
@@ -64,7 +64,7 @@ export const HIGHLIGHT_TIME = 5000
 export const TOAST_NOTIFICATION_DURATION = 10000 // in milliseconds
 
 export const PRICE_ESTIMATION_DEBOUNCE_TIME = 200
-export const PRICE_ESTIMATION_PRECISION = 5
+export const DEFAULT_DECIMAL_PLACES = 5
```

Comment:
Why did you go for a new constant instead of modifying `DEFAULT_DECIMALS`?

Is there places where we want 4 decimals? I understand is more difficult to change the decimals in dex-js cause it affects more places of the app, but for me is a feature that not necessarily needed to be in v1.6

If `DEFAULT_DECIMAL_PLACES` is just a shortcut, I'm ok, and we can include it in this version, but then can we see if it's as easy as modify dex-js  and change the default of 4 to 5? 
It might even be that by doing that, u even change it to 5 in more places that u might have forgotten to add it?

Reply:
i didnt modify yet deFAULT DECIMALS from dex js simply because i was lazy and wanted to do it locally. its just a shortcut as u say

preferably change it later

Reply:
ok, we can try to change it in develop and see if it breaks anything. If it just works, it'll be actually easier than the current approach

In reality, this 4->5 thing i believe would be better to have done it in develop and left it out of v1.6.